src/webgpu/shader/validation/shader_validation_test.ts

```diff
@@ -12,10 +12,8 @@ export class ShaderValidationTest extends GPUTest {
    * ```ts
    * t.expectCompileResult(true, `wgsl code`); // Expect success
    * t.expectCompileResult(false, `wgsl code`); // Expect validation error with any error string
-   * t.expectCompileResult('substr', `wgsl code`); // Expect validation error containing 'substr'
    * ```
-   *
-   * TODO(gpuweb/gpuweb#1813): Remove the "string" overload if there are no standard error codes.
+   * foobar
    */
   expectCompileResult(expectedResult: boolean | string, code: string) {
```

Comment:
Type should change too:

```suggestion
  expectCompileResult(expectedResult: boolean, code: string) {
```

(This may result in type check errors - if any call sites are passing strings)

Reply:
Committed this suggestion, will see if it passes checks.